### PR TITLE
Add tests for `Ellipsoid` and `Hexahedron`

### PR DESCRIPTION
**Pull request by JoshuaLampert**
Integrating `Ellipsoid` and `Hexahedron` should simply work. However, I needed to set pretty high tolerances for `Ellipsoid` for some reason. Do you know why @mikeingold?
I also added other geometries, which do not implement any parametric function yet, in the support matrix, cf. #57

### CODE REVIEW

**Comment by JoshuaLampert** [comment]
Just as a note: The link behind the 🎗️ in the docs doesn't seem to work.

**Comment by JoshuaLampert** [comment]
> However, I needed to set pretty high tolerances for `Ellipsoid` for some reason. Do you know why @mikeingold?

Ah, I see why. The result of `measure` is already off by the tolerance because Meshes.jl doesn't implement an analytical  `measure` for an `Ellipsoid` (which is also not so easy to determine in general) and thus uses a [fallback](https://github.com/JuliaGeometry/Meshes.jl/blob/994b30a17c830a7d9f576af90b252e717dfc0fcb/src/measures.jl#L112).

```julia
julia> using Meshes, MeshIntegrals

julia> origin = Point(0, 0, 0)
Point with Cartesian{NoDatum} coordinates
├─ x: 0.0 m
├─ y: 0.0 m
└─ z: 0.0 m

julia> radii = (1.0, 1.0, 1.0)
(1.0, 1.0, 1.0)

julia> ellipsoid = Ellipsoid(radii, origin)
Ellipsoid
├─ radii: (1.0 m, 1.0 m, 1.0 m)
├─ center: Point(x: 0.0 m, y: 0.0 m, z: 0.0 m)
└─ rotation: LinearAlgebra.UniformScaling{Bool}(true)

julia> using Unitful: m

julia> 4pi*m^2 - measure(ellipsoid)
0.02225157793214194 m^2

julia> 4pi*m^2 - integral(f, ellipsoid, GaussLegendre(100))
-4.96278573791642e-11 m^2
```

The result from `integral` is actually much more precise.

**Comment by mikeingold** [comment]
> Ah, I see why. The result of `measure` is already off by the tolerance because Meshes.jl doesn't implement an analytical  `measure` for an `Ellipsoid` (which is also not so easy to determine in general) and thus uses a [fallback](https://github.com/JuliaGeometry/Meshes.jl/blob/994b30a17c830a7d9f576af90b252e717dfc0fcb/src/measures.jl#L112).
> 
> The result from `integral` is actually much more precise.

Exactly. This is something I've definitely seen. When there's no analytic solution defined for measure, Meshes will `simplexify` the geometry and essentially take a finite element approximation, so integrating a unit integrand function will probably be more accurate. This is one of the motivations behind moving away from the old auto tests that relied on measure as a benchmark.

**Comment by mikeingold** [comment]
> Just as a note: The link behind the 🎗️ in the docs doesn't seem to work.

Weird. It looks like the Documenter site's `stable` version is stale and
hasn't been updated since ~v13.2. The pages under `dev` are up to date,
though. Have you ever seen this kind of thing happen before?

**Comment by JoshuaLampert** [comment]
> Weird. It looks like the Documenter site's `stable` version is stale and
hasn't been updated since ~v13.2. The pages under `dev` are up to date,
though. Have you ever seen this kind of thing happen before?

Indeed. I don't know why this is and I don't see any suspicious in the logs.

**Review by mikeingold — CHANGES_REQUESTED** [review]

**Comment by mikeingold** [comment]
> > Weird. It looks like the Documenter site's `stable` version is stale and
> > hasn't been updated since ~v13.2. The pages under `dev` are up to date,
> > though. Have you ever seen this kind of thing happen before?
> 
> Indeed. I don't know why this is and I don't see anything suspicious in the logs.

We don't have to hold up this PR over it. I just added #103 for tracking.

File: docs/src/supportmatrix.md
[the remark this answers is not in the document]
Comment (JoshuaLampert) [reply docs/src/supportmatrix.md:1]: I pushed two commits addressing these points.

**Review by mikeingold — APPROVED** [review]

**Pull request merged**